NAMESPACE

```diff
@@ -0,0 +1,6 @@
+# Generated by roxygen2: do not edit by hand
+
+export(get_scene_table)
+export(run_app)
+export(view_landsat)
+importFrom(magrittr,"%>%")
```

Comment:
Unless you have a function that gets executed many times the cleanest way to import a function from another package is using `package_name::function_name` in your function instead of adding `import package_name` to the list of imports for that function. I do `importFrom(magrittr,"%>%")` because it would look kind of weird to put `magrittr::%>%` in the code.